lib/common/events.ts

```diff
@@ -54,12 +54,12 @@ export function patchEventTarget(
     // invoke static task.invoke
     task.invoke(task, target, [event]);
   };
-
-  // global shared zoneAwareCallback to handle all event callback with capture = false
-  const globalZoneAwareCallback = function(event: Event) {
+  
+  // global shared zoneAwareCallback to handle all event callback with possible capture
+  const globalZoneAwareCallbackBase = function(event: Event, capture: string) {
```

Comment:
Thanks for the PR, but if we changed the code like this, there will be an additional function call for all callbacks, it will have a little impact for performance.

So I copy the code on purpose, just want to have some better performance.